plugins/cube/cubemap.cpp

```diff
@@ -51,7 +51,7 @@ void wf_cube_background_cubemap::reload_texture()
 
             GL_CALL(glDeleteTextures(1, &tex));
             tex = -1;
-            break;
+            goto render_end;
```

Comment:
I definitely don't see why this thing here is necessary?

Reply:
Ah, I read it as `return` and not `break`. Indeed, it is not needed.